test/test-profiler.ts

```diff
@@ -919,48 +914,4 @@ describe('Profiler', () => {
       }
     );
   });
-  describe('parseBackoffDuration', () => {
```

Comment:
Why are these tests no longer necessary?

Reply:
I removed the tested function which was dead code missed in https://github.com/googleapis/cloud-profiler-nodejs/pull/402